website/content/api-docs/system/internal-counters.mdx

```diff
@@ -89,24 +89,180 @@ $ curl \
+was not capturing data for the months Jan to March for any reason, the response
+will only include information for the months of May and June.
+
+- The response contains the total activity for the billing period and the
+attributions of the total activity against specific components in Vault. This
+helps in understanding the total activity better by knowing which components in
+Vault are leading to that top level activity count. First level of attribution
+breakdowns are by namespaces and months, under the `by_namespaces` and `months`
+JSON block respectively.
+
+```json
+{
+   "by_namespace":[],
+   "months":[],
+}
+```
+
+- `by_namespaces` breakdowns provide attributions of each namespace to the total
+activity count. These namespace attributions are further broken down by mount
+level attributions, wherein information can be found on attributions of each
+mount path within a given namespace to the overall activity of the namespace.
+
+```json
+{
+   "by_namespace":[
+      {
+         "namespace_id":"root",
+         "namespace_path":"",
+         "counts":{},
+         "mounts":[
+            {
+               "path":"auth/up1/",
+               "counts":{}
+            },
+            {
+               "path":"auth/up2/",
+               "counts":{}
+            }
+         ]
+      }
+   ]
+}
+```
+
+- `months` breakdowns provide attributions of each month to the total activity
+count. These month level attributions are further broken down into namespace and mount
+level attributions for each month.
+
+```json
+{
+   "months":[
+      {
+         "timestamp":"2021-05-01T00:00:00Z",
+         "counts":{},
+         "namespaces":[
+            {
+               "namespace_id":"root",
+               "namespace_path":"",
+               "counts":{},
+               "mounts":[
+                  {
+                     "path":"auth/up2/",
+                     "counts":{}
+                  },
+                  {
+                     "path":"auth/up1/",
+                     "counts":{}
+                  }
+               ]
+            },
+            {
+               "namespace_id":"s07UR",
+               "namespace_path":"ns1/",
+               "counts":{},
+               "mounts":[
+                  {
+                     "path":"auth/up1/",
+                     "counts":{}
+                  },
+                  {
+                     "path":"auth/up2/",
+                     "counts":{}
+                  }
+               ]
+            }
+         ],
+         "new_clients":{}
+      }
+   ],
+}
+```
+
+- Each entry in the `months` breakdown contains a `new_clients` block. Within a
+billing period, when a token is first used, regardless of when it was created,
+that token is considered a new client for that billing period. This means that
+all the active clients in the first month of the billing period will be
+considered new clients for that billing period. These tokens could all be
```

Comment:
Suggestions to rewrite:
While these tokens are generated and counted in the previous billing period, they are still considered new clients in the context of the given billing period.

Reply:
Done. Thanks!